docs/intelligentapps/agentbuilder.md

```diff
@@ -88,6 +81,24 @@ To use an existing MCP server, follow these steps:
 4. Select tools from the MCP server if there are multiple tools available.
 5. Enter your prompts and select **Run** to test the connection.
 
+Here is an example of configuring [Filesystem](https://github.com/modelcontextprotocol/servers/tree/main/src/filesystem) server in AI Toolkit:
+1. In the **Tools** section, select **+ MCP Server**, and then select **+ Add server** in the quick pick.
+2. Select **Connect to an Existing MCP Server**
+3. Select **Command (stdio)**
+> [!Note]
+> Some servers use Python runtime and `uvx` command. The process is the same as using `npx` command.
+4. Navigate to [Server instructions](https://github.com/modelcontextprotocol/servers/tree/main/src/filesystem#npx) and locate `npx` section.
+5. Copy `command` and `args` into input box in AI Toolkit. In Filesystem server example, it will be `npx -y @modelcontextprotocol/server-filesystem /Users/junjieli/.aitk/examples`
```

Comment:
```suggestion
5. Copy the `command` and `args` into the input box in AI Toolkit. For the Filesystem server example, it will be `npx -y @modelcontextprotocol/server-filesystem /Users/<username>/.aitk/examples`
```